src/Compilers/Test/Utilities/CSharp/TestOptions.cs

```diff
@@ -83,34 +81,6 @@ public static CSharpParseOptions WithReplaceFeature(this CSharpParseOptions opti
         {
             return options;
         }
-
-        internal static CSharpParseOptions WithExperimental(this CSharpParseOptions options, params MessageID[] features)
```

Comment:
Should we leave this , for other experimental feature options ?

Reply:
We normally tend to delete all unused code. We can always bring it back when we need in future.